migrations/501000_quoteExtension.js

```diff
@@ -39,7 +39,7 @@ exports.up = (knex) => {
         t.bigIncrements('quoteExtensionId').primary().notNullable()
         t.string('quoteId', 36).notNullable().comment('Common ID between the FSPs for the quote object, decided by the Payer FSP')
         t.foreign('quoteId').references('quoteId').inTable('quote')
-        t.bigInteger('quoteResponseId').unsigned().notNullable().comment('The response to the intial quote')
+        t.bigInteger('quoteResponseId').unsigned().nullable().comment('The response to the intial quote')
```

Comment:
Please create a new migration file to make inact this change, and also handle upgrades.